src/seeder/main.cpp

```diff
@@ -444,12 +444,12 @@ extern "C" void *ThreadStats(void *) {
 }
 
 static const std::string mainnet_seeds[] = {
-    "seeder.clashic.cash", "seeder.bitcoincore.zone", "seeder-mainnet.truevisionofsatoshi.com",
+    "seeder.core.cash", "seeder.bitcoincore.zone", "seeder-mainnet.truevisionofsatoshi.com",
```

Comment:
?

Reply:
seeder.clashic.cash is a real Bitcoin Core Seeder Node domain.